src/pages/patientView/patientHeader/PatientHeader.tsx

```diff
@@ -72,7 +73,7 @@ export default class PatientHeader extends React.Component<IPatientHeaderProps,
             >
                 <span className='clinical-spans' id='patient-attributes'>
                     <a href="javascript:void(0)" onClick={()=>this.props.handlePatientClick(patient.id)}>{patient.id}</a>
-                    {getSpanElements(clinicalDataLegacy, 'lgg_ucsf_2014')}
+                    {getSpanElements(clinicalDataLegacy, studyId)}
```

Comment:
might be better to put the studyId on the parent div, but not a big deal.  up to you.

Reply:
oh missed this comment - yes that's a good point